src/main/resources/add-ons-to-index.json

```diff
@@ -3233,6 +3213,23 @@
         "package": "sync"
       }
     },
+    {
+    "uid": "org.openmrs.module.sync2",
+    "type": "OMOD",
+    "name": "Sync 2.0",
+    "maintainers": [
+      {
+        "name": "Tomasz Mueller",
+        "url": "https://github.com/tomaszmueller"
+      }
+    ],
+    "backend": "org.openmrs.addonindex.backend.Bintray",
+    "bintrayPackageDetails": {
+      "owner": "tomaszmueller",
```

Comment:
Hi @tomaszmueller , on further inspection, I realised that the Bintray details may not be accurate. Could you re-check that for me please? 
I believe that https://bintray.com/openmrs/omod/sync2 is the location of the module?
If yes, then the owner is openmrs